src/main/groovy/org/transmartproject/batch/clinical/facts/ClinicalFactsRowSetFactory.groovy

```diff
@@ -99,6 +128,22 @@ class ClinicalFactsRowSetFactory {
         result.addValue concept, getXtrialNodeFor(concept), value
     }
 
+    private ConceptType getConceptTypeFromColumnsFile(ClinicalVariable var) {
+
+        ConceptType conceptType
+        switch (var.conceptType) {
+            case "CATEGORICAL"  : conceptType = ConceptType.CATEGORICAL;
+                break;
+            case "NUMERICAL"    : conceptType = ConceptType.NUMERICAL;
+                break;
+            default             : conceptType = ConceptType.UNKNOWN;
+                break;
```

Comment:
Should throw here. Preferably should be validated during the `readVariables` step rather than this late, when data is already being inserted.